### DISCUSSION TITLE
Failed build with Jekyll

### DISCUSSION THREAD

**Discussion by Wall-wearer** [alshedivat/al-folio#2584] [Q&A] [answered]
Excuse me, I am a beginner at making a Github Page.

I love this theme, but when I fork it to my own repository and active action. Then change the things in `_config.yaml` and press commit. There is an error:

Warning: the running version of Bundler (2.1.4) is older than the version that created the lockfile (2.5.7). We suggest you to upgrade to the version that created the lockfile by running `gem install bundler:2.5.7`.

It seems the code is running under an old bundler version, and this version doesn't have some library that make it failed in the final:

To use retry middleware with Faraday v2.0+, install `faraday-retry` gem
  Liquid Exception: Liquid syntax error (line 64): Unknown tag 'toc' in /_layouts/post.liquid
/usr/local/bundle/gems/liquid-4.0.4/lib/liquid/block.rb:38:in `unknown_tag': Liquid syntax error (line 64): Unknown tag 'toc' 

Can anyone tell me what I did wrong? Thanks

**Comment by george-gca** [comment]
Check our [FAQ](https://github.com/alshedivat/al-folio/blob/master/FAQ.md#my-webpage-works-locally-but-after-deploying-it-fails-to-build-and-throws-unknown-tag-toc-how-do-i-fix-that).

**Reply by Wall-wearer** [reply]
Thank you. It's been done with steps in Q&A, however, there is another issue. The `css` and links are not correct loaded. As meantioned in the Q&A, set `url` & `base_url`. I think I set correctly, maybe, and use another browser that have never open my link. Still doesn't work. would you mind help me the check if I did right? Thanks

**Reply by george-gca** [reply, marked as answer]
I recommend you starting over and making the minimum changes to make it work before anything else. Also I noticed that you changed the name of your main branch, but some workflows are dependent on the name of the branch. For example, the [build site action](https://github.com/alshedivat/al-folio/blob/fc15dd6cc8156f3cff35148c2db81f771e11206a/.github/workflows/deploy.yml#L5) only works on the branch named `master` or `main`, which we expect to be the name of the main branch.

**Reply by Wall-wearer** [reply]
I did rename the name of Branch. However, it follows things in Q&A. It asks me to change the name to `source`, did I misunderstand something?

**Reply by george-gca** [reply]
Where does it recommend that?

**Reply by Wall-wearer** [reply]
Oh, I see what I wrong THANK YOU
